scripts/rageshake.go

```diff
@@ -110,6 +127,20 @@ func main() {
 	// Make sure bugs directory exists
 	_ = os.Mkdir("bugs", os.ModePerm)
 
+	// serve files under "bugs"
+	fs := http.FileServer(http.Dir("bugs"))
+	fs = http.StripPrefix("/api/listing/", fs)
+
+	// set auth if env vars exist
+	usr := os.Getenv("BUGS_USER")
+	pass := os.Getenv("BUGS_PASS")
+	if usr == "" || pass == "" {
+		fmt.Println("BUGS_USER and BUGS_PASS env vars not found. No authentication is running for /api/listing")
+	} else {
+		fs = basicAuth(fs, usr, pass, "Enter username and password")
```

Comment:
It's an arbitrary string which is sent to the client in a popup, which is usually prefixed with "This website says: REALM" hence the request to enter u/p. I don't care what this is called, so I'll change it to whatever.